docs/navigating-compliance-json.md

```diff
@@ -76,7 +76,6 @@ Seeing the diff button alone will **only** show up for devices using JSON compli
 
 > Note: The backend will process and find all compliance rule objects that exist for the device and the create the a temporary backup and intended configurations on the fly. Meaning, when the **diff** action button is clicked all of the processes happens at that time, to render the diff.
```

Comment:
Can you review this whole page, I don't think this is true either anymore.

Reply:
Got that fixed and fixed some of the screenshots.